src/lib/mongoBackend/mongoRegistrationGet.cpp

```diff
@@ -297,10 +108,10 @@ void mongoRegistrationGet
     //
     // Fill in the Registration with data retrieved from the data base
     //
```

Comment:
All the changes in mongoRegistrationGet() will disappear once we revert to `set*()` functions.